docs/io/unified.rst

```diff
@@ -318,6 +316,15 @@ the keywords ``DATE``, ``DATE-*`` (ISO-8601 datetime strings) and the ``MJD-*``
 For more details regarding the FITS time paper and the implementation,
```

Comment:
I disagree about changing the `astropy_native` default for `QTable`:
- It is an API change. If someone is currently reading FITS tables with time columns or date keywords via `QTable`, those will suddenly turn into Time objects.  One should not assume that the only people using QTable are bleeding edge devs that are fine with code breaking.
- The distinction between Table and QTable is already confusing enough.  I have worked *really* hard to emphasize that "the ONLY, ONLY difference between Table and QTable is in the handling of columns with a unit".  Then I repeat that 5 times.  So now we want to add a footnote that, "also, it reads FITS files differently by default".